code/game/machinery/modular_shield.dm

```diff
@@ -388,6 +388,12 @@
 	icon_state = "recharge_port"
 	density = TRUE
 
+	///This var is what determines if performance modules can connect to this module
+	var/allow_boosters = TRUE
+
+	///This var is what determines if we are a booster or a logistic module
+	var/booster = FALSE
```

Comment:
should this be is_booster?

Reply:
youre right ill rewrite all of the variables to use that instead

Reply:
relabeled